Added the new contents of the community/mailing_lists page

javiereguiluz opened the pull request.

fabpot commented on views/en/community/mailing_lists.html.twig:
```diff
@@ -2,32 +2,32 @@
-{% block label 'Email:' %}
-{% block submit 'Subscribe' %}
+{% block discontinued_message %}
+    Symfony Project discontinued all its user mailing lists on June 2015
```
**all** looks too dramatic; it can be removed I think.

fabpot commented on views/en/community/mailing_lists.html.twig:
```diff
@@ -2,32 +2,32 @@
+{% block discontinued_message %}
+    Symfony Project discontinued all its user mailing lists on June 2015
+    (<a href="{{ path('blog_post', { slug: 'symfony-support-stack-overflow-as-the-main-medium' }) }}">Read the announcement</a>).
+    Community support is now provided on StackOverflow.
```
Can we add a link on StackOverflow?

javiereguiluz replied:
Good point! I've added the link ad `<a href="http://stackoverflow.com/?tags=symfony">StackOverflow</a>`. Do you prefer a link to the generic homepage?

xabbuh commented on views/en/community/mailing_lists.html.twig:
```diff
@@ -2,32 +2,32 @@
+{% block symfony2_title %}Symfony2 Users Group{% endblock %}
+{% block symfony2_description %}
+    Community support for the Symfony 2 and 3 versions. Replaced by the
+    <a href="http://stackoverflow.com/?tags=symfony">Symfony tag</a> on StackOverflow.
```
It looks like http://stackoverflow.com/questions/tagged/symfony2 gives better results.

javiereguiluz replied:
Right now that's true. But we're trying to "flip" the tags and make `symfony` the main and best tag for asking/answering Symfony questions.

xabbuh replied:
The tag itself is not so important. But I don't get any valuable output when using the tags URL parameter. So you could use http://stackoverflow.com/questions/tagged/symfony as well.

fabpot replied:
I agree with @xabbuh

javiereguiluz commented:
I've update the StackOverflow URL, so I think this is mergeable now. Thanks.

xabbuh commented:
on line 19 it's still the same

javiereguiluz commented:
I've updated all the rest of URLs. Thanks.

xabbuh commented on views/en/community/mailing_lists.html.twig:
```diff
@@ -2,32 +2,32 @@
+{% block symfony2_title %}Symfony2 Users Group{% endblock %}
+{% block symfony2_description %}
+    Community support for the Symfony 2 and 3 versions. Replaced by the
+    <a href="http://stackoverflow.com/questions/tagged/symfony">Symfony tag</a> on StackOverflow.
```
Should the tag in the URL be uppercased too?

xabbuh commented on views/en/community/mailing_lists.html.twig:
```diff
@@ -2,32 +2,32 @@
-    </p>
+{% block symfonydev_title %}Developers Group{% endblock %}
+{% block symfonydev_description %}
+    Discussions about the development of the Symfony framework itself. Replaced
```
Framework

javiereguiluz commented:
Closing it in favor of #280 because I deleted my previous repository fork. @xabbuh I've addressed your pending comments, except for the tag uppercase (it's not needed because StackOverflow redirects you to `symfony2` tag when using `Symfony` or `symfony`).
